cloud/client.go

```diff
@@ -76,18 +76,21 @@ func (c *Client) RetrieveResourcesByType(
 
 func (c *Client) GetDevices(ctx context.Context, token string, deviceIDs []string, resourceTypes ...string) *grpc.Iterator {
 	auth := pb.AuthorizationContext{AccessToken: token}
```

Comment:
@jkralik, @ondrejtomcik `AuthorizationContext` should be deprecated and removed.

Reply:
I agree with you. It is planned in another task, but at first it must be removed from grpc-gateway.